analytics/tests/test_counts.py

```diff
@@ -180,3 +180,66 @@ def test_active_users_by_is_bot(self):
         self.assertEqual(InstallationCount.objects.count(), 2)
         self.assertFalse(UserCount.objects.exists())
         self.assertFalse(StreamCount.objects.exists())
+
+    def test_messages_sent_by_message_type(self):
+        # type: () -> None
+        property = 'messages_sent:message_type'
+        stat = COUNT_STATS[property]
+
+        user1 = self.create_user('email1')
+        user2 = self.create_user('email2')
+        user3 = self.create_user('email3')
+
+        private_stream1, rec_private_stream1 = self.create_stream_with_recipient(name='private_stream1', invite_only=True)
```

Comment:
Okay! I did this so I didnt' have to set the streams to some throwaway variable:
` public_stream_recipient1 = self.create_stream_with_recipient(name='public_stream1')[1]
`